internal/glfwwin/wglcontext_windows.go

```diff
@@ -208,15 +208,15 @@ func (w *Window) choosePixelFormat(ctxconfig *ctxconfig, fbconfig_ *fbconfig) (i
 func makeContextCurrentWGL(window *Window) error {
 	if window != nil {
 		if err := wglMakeCurrent(window.context.wgl.dc, window.context.wgl.handle); err != nil {
-			_glfw.contextSlot.set(0)
+			_ = _glfw.contextSlot.set(0)
```

Comment:
This should return the error

Reply:
NVM, this cannot

Reply:
so it should only return the error from contextSlot.set, and not from wglMakeCurrent? or should we somehow wrap both errors

Reply:
ah, your comment hadn't loaded. my bad :P